### PR TITLE
ci: add code coverage

### PR DESCRIPTION
**Pull request by evilrobot-01**
Adds code coverage checks.

Coverage report from this branch at https://app.codecov.io/gh/r0gue-io/pop-cli/tree/frank%2Fcode-coverage.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Build & Test Pop-CLI
+name: ci
```
Comment (AlexD10S) [inline .github/workflows/ci.yml:1]: Why this change?

Comment (evilrobot-01) [reply .github/workflows/ci.yml:1]: Better naming for all the CI checks, which are slowly growing to be more than just "build and test". It also keeps the name short, so when viewed in the checks list below it is easier to parse.

**Review by AlexD10S — APPROVED** [review]
